test/gtest/shp/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ add_executable(
   shp-tests.cpp
   ../common/all.cpp
   # need to implement same API as MHP
```

Comment:
```suggestion
```

And you can probably delete this: https://github.com/oneapi-src/distributed-ranges/blob/de9169185e86f33db4e7dd8ff4a3b86453eecc54/benchmarks/gbench/common/distributed_vector.cpp#L79